### DISCUSSION TITLE
[1.1 version] Stability Problem Collection.

### DISCUSSION THREAD

**Discussion by luzhijing** [apache/doris#12134] [A - General / Q&A]
Hi the community,

In order to further improve the stability of Apache Doris , we will collect the stability issues of the latest release version from now on.
If you meet some problems such as Frontend & Backend Crash , Backend Abnormal OOM etc in Apache Doris 1.1 version, please post it in this discussion. We will help you solve the problems , and if it is confirmed as a BUG , we will fix it in the subsequent versions as soon as possible.
If the problem has been fixed in the previous version, we will recommend you to upgrade to the latest version or tell you how to patch to solve the problem.

You can also provide the following information so that we can locate the fault causes faster:

Doris vision: e.g. 1.1.1 Release
Error message:
What operation cause the problem: Query , Load , ect.
Performance:
OS: e.g. Centos , Ubantu or else
Machine configuration: CPU Core , Memory , Storage
Architecture: e.g. x86 or ARM
Whether it can be Repetition or not and Repetition Step:
Relevant Issuees: #XXXXX
——————————

为了进一步提升 Apache Doris 的稳定性，从现在开始，我们将对当前最新 Release 的版本征集稳定性问题。

如果您在 Apache Doris 1.1 版本（包括 1.1.0、1.1.1、1.1.x 等）遇到了诸如 FE&BE挂掉 或 BE 异常OOM 之类的问题，请及时与我们取得联系，我们将会帮您解决问题。如果确认是 Bug 的话，我们将会尽快修复并在后续版本中完成上线，并且有 Apache Doris 周边礼品送出。
如果是之前版本已经修复过的问题，我们将向您推荐升级到最新版本或者告诉您如何打 Patch 来解决这一问题。

为了便于可以更好定位问题，可以提供以下信息：

当前版本：例如 Apache Doris 1.1.1
报错信息：
执行了什么操作导致的报错：比如执行的SQL、导入任务的数据量及频率等
具体表现：
操作系统：
机器配置：CPU核数、内存、磁盘
平台：x86 或 ARM平台
是否能复现以及复现步骤
相关Issue：

**Comment by tkzhu818** [comment]
routineload: kafka_partitions  kafka_offsets 设置时间戳 无效 不能从指定时间开始消费

**Reply by luzhijing** [reply]
> routineload: kafka_partitions kafka_offsets 设置时间戳 无效 不能从指定时间开始消费

Which version?
